bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinBaseHandler.java

```diff
@@ -0,0 +1,202 @@
+                        changePower(((OnOffType) command).equals(OnOffType.ON));
+                        handled = true;
+                    }
+                    break;
+                case DaikinBindingConstants.CHANNEL_AC_TEMP:
+                    if (changeSetPoint(command)) {
+                        handled = true;
+                    }
+                    break;
+                case DaikinBindingConstants.CHANNEL_AIRBASE_AC_FAN_SPEED:
+                case DaikinBindingConstants.CHANNEL_AC_FAN_SPEED:
+                    if (command instanceof StringType) {
+                        changeFanSpeed(((StringType) command).toString());
+                        handled = true;
+                    }
+                    break;
+                case DaikinBindingConstants.CHANNEL_AC_MODE:
+                    if (command instanceof StringType) {
+                        changeMode(((StringType) command).toString());
+                        handled = true;
+                    }
+                    break;
+            }
+            if (!handled && !handleCommandInternal(channelUID, command)) {
+                logger.debug("Received command ({}) of wrong type for thing '{}' on channel {}", command, thing.getUID().getAsString(), channelUID.getId());
+            }
+        } catch (DaikinCommunicationException ex) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, ex.getMessage());
+        }
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing Daikin AC Unit");
+        DaikinConfiguration config = getConfigAs(DaikinConfiguration.class);
+        if (config.host == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Host address must be set");
+        } else {
+            webTargets = new DaikinWebTargets(config.host);
+            refreshInterval = config.refresh;
+
+            schedulePoll();
+        }
+    }
+
+    @Override
+    public void handleRemoval() {
+        super.handleRemoval();
+        stopPoll();
+    }
+
+    @Override
+    public void dispose() {
+        super.dispose();
+        stopPoll();
+    }
+
+    protected void schedulePoll() {
+        if (pollFuture != null) {
+            pollFuture.cancel(false);
+        }
+        logger.debug("Scheduling poll for 1s out, then every {} s", refreshInterval);
+        pollFuture = scheduler.scheduleWithFixedDelay(this::poll, 1, refreshInterval, TimeUnit.SECONDS);
+    }
+
+    protected synchronized void stopPoll() {
+        if (pollFuture != null && !pollFuture.isCancelled()) {
+            pollFuture.cancel(true);
+            pollFuture = null;
+        }
+    }
+
+    private synchronized void poll() {
+        try {
+            logger.debug("Polling for state");
+            pollStatus();
+        } catch (IOException e) {
+            logger.debug("Could not connect to Daikin controller", e);
```

Comment:
Is the stack trace really providing additional information? If not, please remove. The status update is logged anyway.

Reply:
This was also from the previous code. I have removed the logging